crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -97,7 +97,7 @@ def finished?
 
     def save
       progress_file_path.open("w") do |f|
-        f.write(JSON.pretty_generate(progress))
+        f.write(YAML.dump(progress))
```

Comment:
No, it's related because it was necessary to fix this in order to get the test suite working.